webroot/src/models/PrivilegePurchaseOption/PrivilegePurchaseOption.model.spec.ts

```
@@ -14,6 +14,11 @@ import {
 import { State } from '@models/State/State.model';
 import i18n from '@/i18n';
 
+const chaiMatchPattern = require('chai-match-pattern');
+const chai = require('chai').use(chaiMatchPattern);
+
+const { expect } = chai;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'expect'. Consider to delete it or rename it.

'expect' is defined here:


(lint/suspicious/noRedeclare)